mods/player_api/README.txt

```diff
@@ -30,3 +30,6 @@ celeron55, Perttu Ahola <celeron55@gmail.com> (CC BY-SA 3.0):
 sonictechtonic (CC BY 3.0):
 https://www.freesound.org/people/sonictechtonic/sounds/241872/
   player_damage.ogg
+
+An0n3m0us (CC BY 3.0):
```

Comment:
This is best moved to be under:
```
stujones11 (CC BY-SA 3.0):
character.b3d
character.blend -- Both derived from a model by MirceaKitsune (CC BY-SA 3.0)
```
And your license has to be the same license because you have altered something licensed as 'share alike'.

Reply:
Will do.

>And your license has to be the same license because you have altered something licensed as 'share alike'.

Oh, I meant to do that but I forgot.

Reply:
Like this?:
```
stujones11 (CC BY-SA 3.0):
  character.b3d
  character.blend -- Both derived from a model by MirceaKitsune (CC BY-SA 3.0)

An0n3m0us (CC BY-SA 3.0):
  character.b3d
  character.blend -- Player animation improvement
```

Reply:
That's fine for me.

Reply:
OK, done.